StudentsApp/Student.cs

```diff
@@ -67,14 +67,14 @@ public void CompleteTasks(Queue<string> list)
         public void GetBooks()
         {
             Console.WriteLine("What books do you want to take? When list is full press spacebar");
-            bool IsListFinshed = false;
+            bool isListFinshed = false;
             string book = string.Empty;
-            while (!IsListFinshed)
+            while (!isListFinshed)
             {
                 book = Console.ReadLine();
                 if (book == " ")
                 {
-                    IsListFinshed = true;
+                    isListFinshed = true;
                     continue;
                 }
                 if (String.IsNullOrEmpty(book))
```

Comment:
Missed 'this/ for BookList.
can't add a comment there since these changes are not included to the pull request.

Reply:
The same about TaksList.